test/VoteDelegateFactory.t.sol

```diff
@@ -42,18 +42,21 @@ contract VoteDelegateFactoryTest is DssTest {
     }
 
     function testCreate() public {
-        address proxy = factory.getAddress(address(1));
+        uint256 snapshot = vm.snapshot();
+        vm.prank(address(1)); address proxy = factory.create();
+        vm.revertTo(snapshot);
```

Comment:
I think this could be better replaced with:
```
address proxy = vm.computeCreateAddress(address(factory), vm.getNonce(address(factory)));
```
Avoids the snapshotting overhead. We can also remove the `prank` part as not needed and gives a false feeling that the sender has something to do with the resulting address when it is just about the nonce.